src/cid_prefix.rs

```diff
@@ -41,6 +52,37 @@ impl CidPrefix {
         })
     }
 
+    #[allow(dead_code)]
```

Comment:
maybe `cfg(test)` if you don't want to re-use it in future

Reply:
I think it will be used on the server implementation